pages/About/index.js

```diff
@@ -1,13 +1,15 @@
 import Heading from "../../components/about-heading";
 import Informations from "../../components/about-info";
 import Divider from "../../components/about-char-divider";
+import AboutContact from "../../components/about-contact";
 
 function index() {
```

Comment:
@MuhannedNoman what do u think about naming the component with small letters and just `index` instead of the folder name?  this is not recommended, isn't it?